UKHomeOffice/sgmr-service
# SPCR-385 Add time range to arrival/departure

### AC / Description of changes
A user should be able to enter a time rage of 6 hours on the arrival and departure forms.

### To Test
- Create a new voyage
- > You should see inputs with a time range
- Click save and continue
-> All fields should error
- Inout an invalid time e.g. 24:60
- > This should error
- Input a `to` time before the `from` time
- > You should get another error
- Enter a time range greater than 6 hours
- > This should error
- Complete the departure form and continue
- On the arrival form, click save and continue
- > All fields should error
- > Repeat error steps for departure
- Input an arrival date before your departure
- > This should error
- Input the same date as your departure
- Input a `from` time before your departure `from` time
- > This should error
- Input correct details
- Complete the voyage plan
-> It should submit without errors

### Notes

---

**Remember to merge to feature branches not `master`.**
